src/components/navbar/NavBar.jsx

```diff
@@ -0,0 +1,29 @@
+import { Link } from 'react-router-dom';
+import { IoIosArrowBack, IoIosSettings } from 'react-icons/io';
+import { BiMicrophone } from 'react-icons/bi';
+import classes from './NavBar.module.css';
+
```

Comment:
The error with the Navbar module import for styles is still persistent.

![image](https://github.com/haftamudesta/React-Capstone-Project/assets/38088818/d121de80-eb67-4df9-96a9-d58836cfbc70)

- Kindly pay attention to the file names when importing. You are import `NavBar.module.css` instead of `Navbar.module.css`, the only issue is that you are using the capital letter `B` in `NavBar` instead of the small letter `b` for `Navbar`.